functions/sample_function_test.ts

```diff
@@ -0,0 +1,14 @@
+import { SlackFunctionTester } from "deno-slack-sdk/mod.ts";
+import { assertEquals } from "https://deno.land/std@0.153.0/testing/asserts.ts";
+import SampleFunction from "./sample_function.ts";
+
+const { createContext } = SlackFunctionTester("sample_function");
+
+Deno.test("Reverse string function test", async () => {
```

Comment:
Test title is incorrect, but test itself looks good

Reply:
Ah, good catch! 🤦🏼‍♀️